dynamiccv/urls.py

```diff
@@ -23,4 +23,5 @@
     path("ckeditor5/", include('django_ckeditor_5.urls')),
     path('djrichtextfield/', include('djrichtextfield.urls')),
     path('', include('home.urls')),
+    path('profile', include('profiles.urls')),
```

Comment:
Issue (blocking): You just need the `/` at the end of `profile` so that it is `profile/` or else it doesn't seem to work.

Reply:
I'll add it for the other routes, it currently works as we only have a base url view so there's no trailing text after, good catch

Reply:
Fixed